include/LogicBlock/SMTLibLogicModel.hpp

```diff
@@ -0,0 +1,34 @@
+#ifndef LOGICBLOCKS_SMTLIBLOGICMODEL_HPP
+#define LOGICBLOCKS_SMTLIBLOGICMODEL_HPP
```

Comment:
Not sure how this is handled in the rest of the project, but in most of our other projects we now use `#pragma once` instead of header guards.